### PR TITLE
Faster blaster bolts

### PR DESCRIPTION
**Pull request by Pointedstick**
Something that makes Blaster weapons feel unimpressive is the slow speed at which their projectiles travel. They feel more like blaster _blobs_ than blaster _bolts_.

This change increases blaster bolt speed by 50%, keeping the overall range the same. Now the bolts look much more impressive and dangerous (especially when combined with https://github.com/endless-sky/endless-sky/pull/3306). This also helps Blaster turrets get a few more hits in against small fast-moving targets, which should improve their viability compared to Laser turrets.

### CODE REVIEW

File: data/weapons.txt [hunk 1]
```diff
@@ -130,8 +130,8 @@ outfit "Modified Blaster Turret"
 		"hit effect" "blaster impact"
 		"inaccuracy" 4
 		"turret turn" 4
-		"velocity" 7.5
-		"lifetime" 60
+		"velocity" 15
+		"lifetime" 30
```
Comment (Amazinite) [inline data/weapons.txt]: The modified blaster turret has a higher velocity than the modified blaster gun?

**Comment by endless-sky** [comment]
Blaster bolts are *supposed* to be wimpy-looking. Its the smallest, cheapest weapon in the game.

**Comment by Pointedstick** [comment]
I can't say I understand why you want the Blaster to both be and seem so awful. It's like you're telling players not to use it, that it's almost a joke weapon. In which case... why bother including it, and using as the default armament for more than half the human ships--including many Republic warships?

**Comment by KiLEdEnNis** [comment]
I think making it a little bit faster is fine either way though, it won't become that much better but at least using it is a little less bad

**Comment by jafdy** [comment]
While the basic blaster can look that way it seems a shame to resign an entire class of weaponry to uselessness because of this.

**Comment by Amazinite** [comment]
>Blaster bolts are supposed to be wimpy-looking. Its the smallest, cheapest weapon in the game.

I can agree that Blasters should look wimpy and that their bolts should be slow, but I honestly still find the firing heat and energy way too taxing to even justify using blasters over lasers even early game, because the increased firing heat and energy of blasters practically makes you spend more money on other outfits in order to support it that it does to just buy a laser.

**Comment by endless-sky** [comment]
A beam laser is 60% larger than a blaster, only does 33% more shield damage, and has a 30% shorter range. A heavy laser is 200% larger, does 140% more damage, and still has a slightly shorter range. I could certainly see an argument for reducing the heat and energy requirements of the blaster, but making it more powerful doesn't make sense - a 5 T weapon *ought* to be significantly wimpier than an 8 or 15 T one.

(Speaking of which, I also think it would be reasonable to buff the beam laser a bit. Or alternatively, to make the heavy laser and electron beam a bit less powerful.)

**Comment by Pointedstick** [comment]
I don't object to the Blaster's low damage. Here are the ways I think we could make them more attractive:
- Heat and energy could stand to be reduced
- Ships could have a few more gun and turret mounts to fully take advantage of them (https://github.com/endless-sky/endless-sky/issues/3264)
- They could be look and sound more satisfying (this PR and https://github.com/endless-sky/endless-sky/pull/3306)
- Offer a Modified Quad Blaster turret (https://github.com/endless-sky/endless-sky/pull/3319)

**Comment by Amazinite** [comment]
>(Speaking of which, I also think it would be reasonable to buff the beam laser a bit. Or alternatively, to make the heavy laser and electron beam a bit less powerful.)

In what way would they be changed? While I find the damage of the lasers a little high, where they really stand out is their high efficiency of firing heat and energy. Compared to the damage they produce, the human lasers produce among the lowest heat and require among the least energy, sitting up there with the Moonbeam, Bombardment Cannon, and Thrasher Cannon.

By that manner, I'd actually make all three human lasers less powerful by increasing their firing energies and heats by 45%. That might seem like a lot, but currently that only puts the Electron Beam at having just as much firing heat + energy as the modified blaster (except with more of that drain taken up by energy than heat compared to a modified blaster).

If that's too much for many ships that currently use those weapons, then a 20% increase in firing heat and energy but a 10% decrease in damage across the board looks like a viable change to me as well (and might actually be a better change).

**Comment by Pointedstick** [comment]
I like the 45% idea. That would make heat management a much bigger part of the game for many ships and players, which seems appropriate.

**Comment by KiLEdEnNis** [comment]
the problem with that would be having to modify all those variants, and some of them may not even "work" anymore
and if the Navy ships get affected too much we may even need to rebalance all the FW stories

**Comment by Amazinite** [comment]
I've implemented the blaster change in #3350, but a change to the laser weapons would take more time to ensure that everything works smoothly.

**Comment by Lineth** [comment]
I have to disagree with the blaster being just straight-up worse than most weapons. Have it do low damage, by all means, but I think it should have some redeeming quality to make it attractive. I see it's niche being a compact, efficient but low-damage weapon. As it is they seem to just be a waste of time. I did maker a personal mod of the mod quad blaster turret with stats based on the current mod blaster lione and it seems actually competitive with the Hai Pulse turret. This looks like it's a poor man's Plasma Turret, which IMO is not bad. Maybe a another smuggler's bay/custom pirate candidate?

**Comment by endless-sky** [comment]
In beb7f03 I've increased blaster bolt speeds by about 20%.

**Comment by Pointedstick** [comment]
Awesome!

**Pull request closed without merging**